docs/docs/getting-started/ssl.md

```diff
@@ -76,51 +46,49 @@ On port 80, Apache's _alias_ module is used to set a _Redirect permanent_ for th
 
 ```
 <VirtualHost *:443>
-      SSLEngine on
-      SSLProtocol All -SSLv2 -SSLv3
-      SSLCompression off
-      SSLCertificateFile /etc/ssl/certs/ssl-cert-snakeoil.pem
-      SSLCertificateKeyFile /etc/ssl/private/ssl-cert-snakeoil.key
-      Header always set Strict-Transport-Security "max-age=31536000"
-
-      ServerName airtime.example.com
-      #ServerAlias www.example.com
-
-      ServerAdmin admin@example.com
-
-      DocumentRoot /usr/share/airtime/php/legacy/public
-      DirectoryIndex index.php
-
-      <Directory /usr/share/airtime/php/legacy/public>
-              Options -Indexes FollowSymLinks MultiViews
-              AllowOverride all
-              Order allow,deny
-              Allow from all
-      </Directory>
+    SSLEngine on
+    SSLProtocol All -SSLv2 -SSLv3
+    SSLCompression off
+    SSLCertificateFile /etc/ssl/certs/ssl-cert-snakeoil.pem
+    SSLCertificateKeyFile /etc/ssl/private/ssl-cert-snakeoil.key
+    Header always set Strict-Transport-Security "max-age=31536000"
+
+    ServerName airtime.example.com
+    #ServerAlias www.example.com
+
+    ServerAdmin admin@example.com
+
+    DocumentRoot /usr/share/airtime/php/legacy/public
+    DirectoryIndex index.php
+
+    <Directory /usr/share/airtime/php/legacy/public>
+        Options -Indexes FollowSymLinks MultiViews
+        AllowOverride all
+        Order allow,deny
+        Allow from all
+    </Directory>
 </VirtualHost>
 
 <VirtualHost *:80>
-      ServerName airtime.example.com
+    ServerName airtime.example.com
 
-      ServerAdmin admin@example.com
+    ServerAdmin admin@example.com
 
-      DocumentRoot /usr/share/airtime/php/legacy/public
-      Redirect permanent /login https://airtime.example.com/login
+    DocumentRoot /usr/share/airtime/php/legacy/public
+    Redirect permanent /login https://airtime.example.com/login
 
-      SetEnv APPLICATION_ENV "production"
+    SetEnv APPLICATION_ENV "production"
 
-      <Directory /usr/share/airtime/php/legacy/public>
-              Options -Indexes FollowSymLinks MultiViews
-              AllowOverride All
-              Order allow,deny
-              Allow from all
-      </Directory>
+    <Directory /usr/share/airtime/php/legacy/public>
+        Options -Indexes FollowSymLinks MultiViews
+        AllowOverride All
+        Order allow,deny
+        Allow from all
+    </Directory>
 </VirtualHost>
 ```
```

Comment:
- I think the codeblocks support apache syntax highlighting
- Can we import code blocks from files ? I would like to move this to a dedicated file.